azdev/operations/extensions/__init__.py

```diff
@@ -291,12 +291,12 @@ def publish_extensions(extensions, storage_account, storage_account_key, storage
     subheading('Uploading WHLs')
     for whl_path in whl_files:
         whl_file = os.path.split(whl_path)[-1]
+        from azure.storage.blob import BlockBlobService
+        client = BlockBlobService(account_name=storage_account, account_key=storage_account_key)
+        exists = client.exists(container_name=storage_container, blob_name=whl_file)
```

Comment:
L296 can be moved into the not yes branch.

Reply:
If L296 and L297 moving into not yes branch, it means no change

Reply:
If L296 moves to not yes branch, there is no client for exists().